### ISSUE TITLE
support mime v 2.0.0

### ISSUE THREAD

**Issue by hatemragab** [781flyingdutchman/background_downloader#409]
```
Resolving dependencies...
Because background_downloader 8.6.0 depends on mime ^1.0.1 and no versions of background_downloader match >8.6.0 <9.0.0, background_downloader ^8.6.0 requires mime ^1.0.1.
And because every version of v_chat_sdk_core from path depends on background_downloader ^8.6.0, every version of v_chat_sdk_core from path requires mime ^1.0.1.
So, because super_up_core depends on both mime ^2.0.0 and v_chat_sdk_core from path, version solving failed.
BootstrapException: Failed to install.: super_up_core at /Users/hatemragap/flutter_projects/superup/packages/super_up_core.
```

**Comment by 781flyingdutchman** [comment]
Thanks for letting me know - I can upgrade it in a future version, but for now just add a dependency override to your pubspec.yaml:
```
dependency_overrides:
  mime: '2.0.0'
```

**Comment by 781flyingdutchman** [comment]
Implemented in 8.7.0

**Comment by hatemragab** [comment]
thanks
